common/src/main/java/com/wynntils/utils/colors/CommonColors.java

```diff
@@ -16,6 +16,7 @@ public final class CommonColors {
     public static final CustomColor BROWN = CustomColor.fromInt(0x563100).withAlpha(255);
     public static final CustomColor PURPLE = CustomColor.fromInt(0xb200ff).withAlpha(255);
     public static final CustomColor CYAN = CustomColor.fromInt(0x438e82).withAlpha(255);
+    public static final CustomColor AQUA = CustomColor.fromInt(0x00ffff).withAlpha(255);
```

Comment:
I have just added:
```
    public static final CustomColor AQUA = CustomColor.fromInt(0x00cccc).withAlpha(255);
```

in my mapdata branch, for how the province place labels where colored. That is apparently a darker shade of aqua. Do we try to converge on one value, or should I rename mine DARK_AQUA?

Reply:
I am happy with how close this version is to the Wynn AQUA color, so can you perhaps rename yours?

Reply:
Okidoki.